modules/main.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
- [OPTIONAL] Linters are tools that will scan your code with a set of style rules and will report any errors to you that they find. In some cases, they can even auto-fix the errors! Instead of disabling `eslint`, it will be nice to fix the issue.